_episodes/12-cluster.md

```diff
@@ -23,143 +27,278 @@ On the first day, her supervisor greets her friendly and welcomes her to the job
+{: .callout}
 
+Lola is asked to use a UNIX command called `ls` (for list directory contents) to have a look around. 
 
-Very often, many users are tempted to think of a high-performance computing installation as one
-giant, magical machine. Sometimes, people will assume that the computer they've logged onto is the
-entire computing cluster. So what's really happening? What computer have we logged on to? The name
-of the current computer we are logged onto can be checked with the `hostname` command. (Clever users
-will notice that the current hostname is also part of our prompt!)
+~~~ 
+$ ls
+~~~
+{: .language-bash}
 
-```
-[remote]$ hostname
-```
-{: .bash}
-```
-gra-login3
-```
+~~~ 
+~~~
 {: .output}
 
-Individual computers that compose a cluster are typically called *nodes* (although you will also
-hear people call them *servers*, *computers* and *machines*). On a cluster, there are different
-types of nodes for different types of tasks. The node where you are right now is called the *head
-node*, *login node* or *submit node*. A login node serves as an access point to the cluster. As a
-gateway, it is well suited for uploading and downloading files, setting up software, and running
-quick tests. It should never be used for doing actual work.
-
-The real work on a cluster gets done by the *worker* (or *execute*) *nodes*. Worker nodes come in
-many shapes and sizes, but generally are dedicated to doing all of the heavy lifting that needs
-doing.
-
-All interaction with the worker nodes is handled by a specialised piece of software called a
-scheduler (the scheduler used in this lesson is called SLURM). We'll learn more about how to use the
-scheduler to submit jobs next, but for now, it can also tell us more information about the worker
-nodes.
-
-For example, we can view all of the worker nodes with the `sinfo` command.
-
-```
-[remote]$ sinfo
-```
-{: .bash}
-```
-PARTITION AVAIL  TIMELIMIT  NODES  STATE NODELIST
-compute*     up 7-00:00:00      1 drain* gra259
-compute*     up 7-00:00:00     11  down* gra[8,99,211,268,376,635,647,803,852,966,986]
-compute*     up 7-00:00:00      1   drng gra272
-compute*     up 7-00:00:00     31   comp gra[988-991,994-1002,1006-1007,1015,1017,1021-1022,1028-...
-compute*     up 7-00:00:00     33  drain gra[225-251,253-256,677,1026]
-compute*     up 7-00:00:00    323    mix gra[7,13,25,41,43-44,56,58-77,107-108,112-113,117,125-12...
-compute*     up 7-00:00:00    464  alloc gra[1-6,9-12,14-19,21-24,26-40,42,45-55,57,100-106,109-1...
-compute*     up 7-00:00:00    176   idle gra[78-98,123-124,128-162,170-172,285-299,429-447,449-45...
-compute*     up 7-00:00:00      3   down gra[20,801,937]
-```
+To no surprise, there is nothing displayed. To prove, that Lola is really logged in to another machine, Lola issues a command to see on what machine she currently is on.
```

Comment:
> To no surprise

That feel dismissive to me, it may be a surprise to some, it may also  not be the case on all supercomputer that mount things like data/ scratch/ ....

Reply:
good point. Thanks. I'll remove that.